packages/flutter/lib/src/services/text_editing.dart

```diff
@@ -195,4 +195,15 @@ class TextSelection extends TextRange {
     affinity.hashCode,
     isDirectional.hashCode
   );
+
+  /// Creates a new [TextSelection] based on the current selection, with the
+  /// provided parameters overridden.
+  TextSelection copyWith({int baseOffset, int extentOffset}) {
```

Comment:
why only those two instead of all four?

nit: spaces around named argument declarations